include/inexor/vulkan-renderer/frame_graph.hpp

```diff
@@ -213,10 +214,10 @@ class PhysicalResource {
 protected:
     // TODO(): Add OOP device functions (see above todo) and only store a wrapper::Device here
     VmaAllocator m_allocator;
-    VkDevice m_device;
+    wrapper::Device &m_device;
```

Comment:
I feel like all the device handles in the physical resource classes should stay as `VkDevice`s, at least for now, as right now it's just convoluting the code.

Reply:
Done.